### ISSUE TITLE
Style change causes @font-face files to be re-requested from server 

### ISSUE THREAD

**Issue by jakelazaroff** [modulz/stitches#587]
# Bug report

## Describe the bug

Whenever styles change, the browser re-requests any fonts loaded with `@font-face`.

## To Reproduce

Minimal test case here: https://codesandbox.io/s/lively-tdd-v09pj

It's easiest to tell if you open the preview in its own tab/window and have the network inspector open when you load the page. That way, you can see the initial request(s) for font files alongside the subsequent one when clicking the button in the preview.

## Expected behavior

The browser should not request fonts it has already downloaded in the same page load.

## System information

- OS: macOS
- Browser (if applies): Firefox, Chrome, probably others
- Version of Stitches: 0.1.9

## Additional context

It seems like styled-components ran into this same issue (https://github.com/styled-components/styled-components/issues/1593). They considered splitting out `@font-face` rules into their own style tag, but ultimately decided to encourage people to set cache headers rather than addressing it (https://github.com/styled-components/styled-components/issues/1593#issuecomment-508748098).

Not sure whether or not y'all will want to make the same tradeoff, but bringing it to your attention anyway.

**Comment by peduarte** [comment]
@jakelazaroff thanks for that, we'll investigate.

**Comment by peduarte** [comment]
Fixed in `^0.2.0`

demo: https://codesandbox.io/s/tvzpz

```bash
npm i @stitches/react@latest
```

**Comment by gajus** [comment]
I've just spent an hour debugging what seemed to be a related issue. However, it turned out it was caused by use of ref preload.

```html
<link
  as='font'
  crossOrigin='anonymous'
  href='/fonts/segoe-ui-regular.woff2'
  rel='preload'
  type='font/woff2'
/>
```

For whatever reason presence of preload was causing fonts to flicker any time Stitches regenerates global styles.

I didn't find a solution, just removed preload tag.
